23-07-workshop/in-progress/query_region_plot_HR_plot_lc.ipynb

```diff
@@ -0,0 +1,1368 @@
+{
```

Comment:
<p>Not all will have HLSP's.</p><p> </p><p>Reference other TIKEBook notebooks; do a markdown link with ../02-whatever, will be able to find it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>